packages/core/src/controllers/SendController.ts

```diff
@@ -103,6 +118,18 @@ export const SendController = {
       this.state.gasPrice &&
       this.state.token?.quantity.decimals
     ) {
+      EventsController.sendEvent({
```

Comment:
Why there is two `SEND_INITIATED` events here?

Reply:
One is for native token and the other one for `erc-20`